client/app/components/group/_pickupEditCreate/pickupEditCreate.controller.js

```diff
@@ -137,4 +137,14 @@ class pickupEditCreateController {
   }
 }
 
+  noPastDateCalender() {
```

Comment:
maybe rather 'setMinDate'? It describes more accurately what the code is doing.

Also, you need to invoke this method somewhere. I'd recommend the $onInit method (just append it)